db/seeds/schedules.rb

```diff
@@ -2,65 +2,17 @@
 
 ecf_september_standard_2021 = Finance::Schedule.find_or_create_by!(name: "ECF September standard 2021")
 [
-  { name: "Output 1 - Participant Start", milestone_date: Date.new(2021, 10, 31), payment_date: Date.new(2021, 11, 30) },
-  { name: "Output 2 – Retention Point 1", milestone_date: Date.new(2022, 1, 31), payment_date: Date.new(2022, 2, 28) },
-  { name: "Output 3 – Retention Point 2", milestone_date: Date.new(2022, 4, 30), payment_date: Date.new(2022, 5, 31) },
-  { name: "Output 4 – Retention Point 3", milestone_date: Date.new(2022, 9, 30), payment_date: Date.new(2022, 10, 31) },
-  { name: "Output 5 – Retention Point 4", milestone_date: Date.new(2023, 1, 31), payment_date: Date.new(2023, 2, 28) },
-  { name: "Output 6 – Participant Completion", milestone_date: Date.new(2023, 4, 30), payment_date: Date.new(2023, 5, 31) },
+  { name: "Output 1 - Participant Start", start_date: Date.new(2021, 9, 1), milestone_date: Date.new(2021, 10, 31), payment_date: Date.new(2021, 11, 30) },
+  { name: "Output 2 – Retention Point 1", start_date: Date.new(2021, 11, 1), milestone_date: Date.new(2022, 1, 31), payment_date: Date.new(2022, 2, 28) },
+  { name: "Output 3 – Retention Point 2", start_date: Date.new(2022, 2, 1), milestone_date: Date.new(2022, 4, 30), payment_date: Date.new(2022, 5, 31) },
+  { name: "Output 4 – Retention Point 3", start_date: Date.new(2022, 5, 1), milestone_date: Date.new(2022, 9, 30), payment_date: Date.new(2022, 10, 31) },
+  { name: "Output 5 – Retention Point 4", start_date: Date.new(2022, 10, 1), milestone_date: Date.new(2023, 1, 31), payment_date: Date.new(2023, 2, 28) },
+  { name: "Output 6 – Participant Completion", start_date: Date.new(2023, 2, 1), milestone_date: Date.new(2023, 4, 30), payment_date: Date.new(2023, 5, 31) },
 ].each do |hash|
   Finance::Milestone.find_or_create_by!(
```

Comment:
has this already been run?

Reply:
Pretty sure not, I will check before running it.